package.json

```diff
@@ -57,10 +57,12 @@
       "<rootDir>/node_modules/(?!lit-html|franklin-sites|p-map|aggregate-error|clean-stack).+\\.js"
     ],
     "testPathIgnorePatterns": [
-      "/__mocks__/"
+      "/__mocks__/",
+      "/.yalc/"
```

Comment:
I'm not sure about adding a reference to `yalc` in package.json if it's not a dependency. I don't think it's a problem to have it in `.gitignore` however.

Reply:
I wasn't sure either but I didn't know how else to handle this issue. Without including `.yalc` in `testPathIgnorePatterns` and `coveragePathIgnorePatterns` I observer errors. Any ideas to avoid its inclusion? Should we remove the `.yalc` references from `package.json` to force devs to remove any yalc stuff themselves (`yalc remove my-package`)?

Reply:
Where do the .yalc files reside? I find it weird that `testPathIgnorePatterns` and `coveragePathIgnorePatterns` ignores other hidden directories and files (`.husky`, `.vscode` etc...)

Reply:
> ignores other hidden directories and files (.husky, .vscode etc...)

Might be because they don't have javascript files within

Reply:
Might be!
In that case, have you tried making `transform` more specific by adding the `src/` path in the regex if that's possible?

Reply:
Or a more generic `transformIgnorePatterns` to ignore directories at the root starting with "."
Maybe `<rootDir>\\..+\\.js`?

Reply:
That's a good idea. Latest commit does this.